src/OrchardCore.Modules/OrchardCore.Alias/Models/AliasPart.cs

```diff
@@ -4,6 +4,9 @@ namespace OrchardCore.Alias.Models
 {
     public class AliasPart : ContentPart
     {
+        // Maximum length that MySql can support in an index under utf8 collation.
+        public const int MaxAliasLength = 767;
```

Comment:
Not here. 
I understand why you want to move it, but on the part it becomes part of the JSON I suspect. Plus is also in the views.

`Startup`  maybe as a central spot?